src/main/java/duke/tasks/TodoTask.java

```diff
@@ -0,0 +1,22 @@
+package duke.tasks;
+
+/**
+ * Represents tasks with no deadline.
+ */
+public class TodoTask extends Task {
+
+    /**
+     * Constructs TodoTask.
+     */
+    public TodoTask(String name, boolean done) {
+        super(name, done);
+    }
+
+    public String getType() {
+        return "T";
+    }
+    @Override
+    public String toString() {
+        return " [T]" + super.toString();
+    }
+}
```

Comment:
Hi, I looked through your code and was quite impressed. There was no coding standard violations as far as I could tell, and I liked that you abstracted the command object which I did not do myself. Good work!

Reply:
Noted, thank you cedric !!